src/commands/dev-env-sync-sql.js

```diff
@@ -197,6 +202,7 @@ export class DevEnvSyncSQLCommand {
 			await this.runImport();
 			console.log( `${ chalk.green( '✓' ) } SQL file imported` );
 		} catch ( err ) {
+			await this.track( 'sql_import_error', { errorMessage: err?.message } );
```

Comment:
```suggestion
			await this.track( 'import_error', { errorMessage: err?.message } );
```

otherwise it'll become `dev_env_sync_sql_sql_import_error`

Reply:
Huh, good catch!